sync_ci/pkg/command/sync_ci.go

```diff
@@ -22,13 +26,67 @@ func (*SyncCICommand) Usage() string {
 }
 
 func (s *SyncCICommand) SetFlags(f *flag.FlagSet) {
-	f.StringVar(&s.Dsn, "dsn", "root:@tcp(127.0.0.1:3306)/sync_ci_data", "dsn")
+	f.StringVar(&s.Dsn, "dsn", "root:@tcp(127.0.0.1:3306)/sync_ci_data", "CI Database dsn")
+	f.StringVar(&s.CaseDsn, "cs", "root:@tcp(127.0.0.1:3306)/issue_case", "Case-issues Database dsn")
+	f.StringVar(&s.GithubDsn, "gh", "root:@tcp(127.0.0.1:3306)/issues", "Github Issues Database dsn")
+	f.StringVar(&s.GithubToken, "tk", "", "Github token to automatically create issues")
+	f.Int64Var(&s.UpdateInterval, "ui", 3600, "The interval (secs) to update")
 	f.StringVar(&s.Port, "port", "36000", "http service port")
 	f.StringVar(&s.LogPath, "lp", "log", "log path")
 	f.StringVar(&s.RulePath, "rp", "pkg/parser/envrules.json", "env rule file path")
 }
 
 func (s *SyncCICommand) Execute(ctx context.Context, f *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
 	server.NewServer(&s.Config).Run()
+	go RunCaseIssueRoutine(s.Config, false)
 	return subcommands.ExitSuccess
 }
+
+func RunCaseIssueRoutine(cfg model.Config, test bool) {
+	if err := model.InitLog(cfg.LogPath); err != nil {
+		log.S().Fatalf("init log error , [error]", err)
+	}
+
+	for {
+		inspectStart := time.Now().Add(-detect.PrInspectLimit)
+		recentStart := time.Now().Add(-time.Duration(cfg.UpdateInterval) * time.Second)
+		cases, err := detect.GetCasesFromPR(cfg, recentStart, inspectStart, false)
+		if err != nil {
```

Comment:
if the err is not nil，the cases is nil according to the completion of the function. It should check it when using the cases below.